packages/acceptance-tests/pkg-tests-core/sources/utils/tests.ts

```diff
@@ -729,13 +729,15 @@ export const generatePkgDriver = ({
           // So add env variable TEST_IGNORE_TIMEOUT_FAILURES to turn on this suppression
           // TODO: investigate whether this is still needed.
           if (process.env.TEST_IGNORE_TIMEOUT_FAILURES) {
+            let timer;
```

Comment:
```suggestion
            let timer: NodeJS.Timeout | undefined;
```

Reply:
Since it passed the type checking, I neglected it. But u r right